deny.toml

```diff
@@ -13,6 +13,10 @@ multiple-versions = "deny"
 skip = [
     { name = "syn", version = "1.0.109" },
     { name = "syn", version = "2.0.23" },
+    { name = "bitflags", version = "1.3.2" },
```

Comment:
can we update these three to include all of the versions including the most recent one? this will make sure that if any of them change, we'll get a warning about what needs to be updated.